src/components/common/ProfilePanel.test.js

```diff
@@ -102,40 +100,40 @@ describe('ProfilePanel', () => {
     expect(component.find('.panel-name-text').text()).toBe('Mark');
 
     // hover over the panel name
-    expect(component.state().nameIsHovering).toBe(false);
+    expect(component.find('.edit-icon-image').exists()).toBe(false);
     component.find('.panel-name').simulate('mouseenter');
-    expect(component.state().nameIsHovering).toBe(true);
+    expect(component.find('.edit-icon-image').exists()).toBe(true);
 
     // click the pencil icon (changes to input)
-    expect(component.state().editingName).toBe(false);
+    expect(component.find('.panel-edit-container').exists()).toBe(false);
     component.find('.edit-icon-image').simulate('click');
-    expect(component.state().editingName).toBe(true);
+    // expect(component.state().editingName).toBe(true);
     expect(component.find('.panel-edit-container')).toHaveLength(1);
 
     // check the input value starts as 'Mark', type in the input 'Not Mark',
     // check that the input value and state changes to 'Not Mark'
-    expect(component.state().name).toBe('Mark');
+    // expect(component.state().name).toBe('Mark');
     expect(component.find('.panel-edit').props().value).toBe('Mark');
     component.find('.panel-edit').simulate('change', {
       target: {
         value: 'Not Mark',
       },
     });
     expect(component.find('.panel-edit').props().value).toBe('Not Mark');
-    expect(component.state().name).toBe('Not Mark');
+    // expect(component.state().name).toBe('Not Mark');
```

Comment:
Note: these are imperative tests, but it's I'm fine with that -- just letting you know
```suggestion
```